modules/nodes-pods-autoscaling-custom-creating-job.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-creating-job_{context}"]
+= Adding a custom metrics autoscaler to a job
+
+You can create a custom metrics autoscaler (CMA) for any `Job` object.
```

Comment:
ScaledJob Spec should be here: https://keda.sh/docs/2.7/concepts/scaling-jobs/#scaledjob-spec